packages/fetchai/agents/tac_controller_contract/aea-config.yaml

```diff
@@ -25,7 +25,7 @@ skills:
 - fetchai/tac_control:0.23.0
 - fetchai/tac_control_contract:0.25.0
 default_connection: fetchai/p2p_libp2p:0.24.0
-default_ledger: ethereum
+default_ledger: fetchai
```

Comment:
Can we please keep the old one and add a new one?

Reply:
I do not think it is justified to introduce another identical agent with only configuration changes. 
We can easily switch between Ethereum and Fetchai Tac by changing two config fields.

Reply:
OK, please add the two test classes then